en/manage-prometheus-metrics.md

```diff
@@ -0,0 +1,96 @@
+Title: Prometheus metrics
+TODO:
+table_of_contents: True
+
+# Prometheus metrics
+
+MAAS services can provide [Prometheus][prometheus] endpoints for collecting
+performance metrics.
+
+These include:
+
+- TFTP server file transfer latency
+- HTTP requests latency
+- Websocket requests latency
+- RPC calls (beween MAAS services) latency
+- Per request DB queries counts
+
+All available metrics are prefixed with `maas_`, to make it easier to look them
+up in Prometheus and Grafana UIs.
+
+## Enabling Prometheus endpoints
+
+Prometheus endpoints are exposed over HTTP by the `rackd` and `regiond`
+processes under the default `/metrics` path, whenever the
+`python3-prometheus-client` library is installed.
```

Comment:
Should clarify that with the MAAS snap it woks out of the box.

Probably provide the apt command to run for the packaged version of MAAS.

Reply:
done